modelarmor/api/ModelArmor.Samples/CreateTemplateWithBasicSdp.cs

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+// [START modelarmor_create_template_with_basic_sdp]
+using System;
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.ModelArmor.V1;
+
+namespace ModelArmor.Samples
```

Comment:
We don't use namespaces in samples because it adds noise and an extra index. Please take a look at KMS samples set as the canonical example.